include/cuda/std/detail/libcxx/include/__config

```diff
@@ -792,6 +792,11 @@ typedef __char32_t char32_t;
 #define _LIBCUDACXX_HAS_NO_ASAN
 #endif
 
+// GCC implemented P0030R1 in 7.0, maybe only available under experimental?
+#if _GNUC_VER < 700
+#define _LIBCUDACXX_NO_HOST_CPP17_HYPOT
```

Comment:
I can see _a_ use of this in cmath that's being removed in this PR. Is this still needed?

Reply:
Removed!